### ISSUE TITLE
OGG is not a supported file type with iOS

### ISSUE THREAD

**Issue by Kenoshen** [bitDecayGames/Helm#230]
Unfortunately, according to the second thing on the list at this ticket: https://github.com/libgdx/libgdx/issues/1828

There is no support on iOS for OGG.  So, for iOS, maybe just create a .wav copy of the assets and then use those instead?

**Comment by MondayHopscotch** [comment]
Yeah, likely I'll just have to dump out .wav versions of all of the assets. Not sure if it makes sense to just move everything over to .wav or try to have android use .ogg still.

I'll address this as I get in a position to work on the iOS version.
